astropy/units/tests/test_equivalencies.py

```diff
@@ -770,3 +771,21 @@ def test_littleh():
     # Then M - 5*log_10(h)  = M + 5 = 17
     withlittlehmag = 17 * (u.mag + u.MagUnit(u.littleh**2))
     assert_quantity_allclose(withlittlehmag.to(u.mag, u.with_H0(H0_10)), 12*u.mag)
+
+
+def test_equivelency():
+    ps = u.pixel_scale(10*u.arcsec/u.pix)
+    assert isinstance(ps, Equivalency)
+    assert isinstance(ps.name, list)
+    assert len(ps.name) == 1
+    assert ps.name[0] == "pixel_scale"
+    assert isinstance(ps.kwargs, list)
+    assert len(ps.kwargs) == 1
+    assert ps.kwargs[0] == dict({'pixscale': 10*u.arcsec/u.pix})
+
+def test_add_equivelencies():
+    e1 = u.pixel_scale(10*u.arcsec/u.pixel) + u.temperature_energy()
+    assert isinstance(e1, Equivalency)
+    assert e1.name == ["pixel_scale", "temperature_energy"]
+    assert isinstance(e1.kwargs, list)
+    assert e1.kwargs == [dict({'pixscale': 10*u.arcsec/u.pix}), dict()]
```

Comment:
Could you add a test of adding an `Equivalency` to a plain list (can be the simplest form of equivalency, say `[u.count, u.bit]`), and check that the result is a plain list?

Reply:
Ah, I added it to the test file in io.asdf. It makes sense to be here. I'll move it here.
Thanks for the review.